robyn/router.py

```diff
@@ -84,20 +85,26 @@ def get_routes(self) -> List[Route]:
 class MiddlewareRouter(BaseRouter):
     def __init__(self) -> None:
         super().__init__()
-        self.routes = []
+        self.global_middlewares: Dict[MiddlewareType, List[FunctionInfo]] = {
```

Comment:
I have a proposition - do you think the code will become simpler if we have two different variables for `global_before_request` and `global_after_requests`?

Reply:
In my opinion this would make the code more verbose, I would prefer to keep only one variable embedding the `before` or `after` information :slightly_smiling_face:

Reply:
Cool cool. Makes sense